PersonaBarPageHeader/package.json

```diff
@@ -1,37 +1,49 @@
 {
   "name": "dnn-persona-bar-page-header",
-  "version": "0.0.29",
+  "version": "0.0.31",
```

Comment:
I keep looking at those versions and I'm getting the impression that we're not really using server correctly?